### PR TITLE
Prepare for cmcc 12

### PR DESCRIPTION
**Pull request by uwilken**
Update libs

tested with master and 2310

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -17,22 +17,22 @@ jobs:
     steps:
       - name: Checkout PR
         if: github.event_name == 'pull_request'
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4.1.1
```
Comment (pitgrap) [inline .github/workflows/build.yml:20]: same as in https://github.com/CoreMedia/content-hub-adapter-rss/pull/12#pullrequestreview-1811070817

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -17,22 +17,22 @@ jobs:
...
         with:
           ref: ${{ github.event.pull_request.head.ref }}
       - name: Checkout
         if: github.event_name != 'pull_request'
         uses: actions/checkout@v3
       - name: Set up JDK 11
-        uses: actions/setup-java@v2
+        uses: actions/setup-java@v4.0.0
```
Comment (pitgrap) [inline .github/workflows/build.yml:27]: and here

**Review by pitgrap — APPROVED** [review]

**Review by pitgrap — CHANGES_REQUESTED** [review]
still one version change in github action missing.

Also missing:
- Changelog for 2.1.0
- change the version to 2310 in README icon

**Comment by uwilken** [comment]
> * change the version to 2310 in README icon

Depends on the meaning of the badge. Does it mean the current or the minimum cms version?

**Comment by pitgrap** [comment]
🤷‍♂️

**Review by mkleine — APPROVED** [review]
lgtm

**Pull request merged**